Pipfile

```diff
@@ -15,14 +15,16 @@ Fabric = "==1.14.1"
 grpcio-tools = "==1.62.2"
 gunicorn = "*"
 isort = "*"
-nodeenv = "==1.0.0"
+mypy-protobuf = "*"
```

Comment:
Can we pin these to a particular version?

Reply:
Sure! Done.